learningresources/models.py

```diff
@@ -32,17 +32,17 @@ class LearningObject(models.Model):
     path_xml = models.TextField()
     mpath = models.TextField()
```

Comment:
There was a comment about renaming this, but the conversation was lost when this file changed. To what are we renaming the `mpath` field?